src/components/EventCalender/Header/EventHeader.tsx

```diff
@@ -58,13 +58,15 @@ function eventHeader({
       data-testid="calendarEventHeader"
     >
       <div className={styles.calendar__header}>
-        <SearchBar
-          placeholder={t('searchEventName')}
-          onSearch={(term) => console.log(`Search term: ${term}`)}
-          inputTestId="searchEvent"
-          buttonTestId="searchButton"
-        />
-        <div className={styles.space}>
+        <div className={styles.calendar__search}>
+          <SearchBar
+            placeholder={t('searchEventName')}
+            onSearch={(term) => console.log(`Search term: ${term}`)}
```

Comment:
1. This was recommended to be updated by Code Rabbit. 
2. Address all the console.log concerns in this file
    - https://github.com/PalisadoesFoundation/talawa-admin/pull/4737#issuecomment-3583461495